deployment/deploy/2030-deploy-ReplaceLoanTokenBeacon.js

```diff
@@ -0,0 +1,409 @@
+            await sendWithMultisig(
+                multisigDeployment.address,
+                loanTokenLogicBeaconWRBTCDeployment.address,
+                data,
+                deployer
+            );
+            log(
+                col.bgBlue(
+                    `>>> DONE. Requires Multisig (${multisigDeployment.address}) signatures to execute tx <<<`
+                )
+            );
+        } else {
+            log(
+                col.bgYellow(
+                    "Skipping LoanTokenLogicWrbtc registration in LoanTokenLogicBeaconWRBTC..."
+                )
+            );
+        }
+
+        /** 4. Registering Loan Protocol Settings Module to LoanTokenLogicBeaconWrbtc */
+        activeModuleIndex = await loanTokenLogicBeaconWRBTCContract.activeModuleIndex(
+            ethers.utils.formatBytes32String(modulesList.LoanTokenSettingsLowerAdmin)
+        );
+        moduleImplAddress = await loanTokenLogicBeaconWRBTCContract.moduleUpgradeLog(
+            ethers.utils.formatBytes32String(modulesList.LoanTokenSettingsLowerAdmin),
+            activeModuleIndex
+        );
+        if (
+            moduleImplAddress["implementation"].toLowerCase() !==
+            loanTokenSettingsLowerAdminDeployment.address.toLowerCase()
+        ) {
+            log(
+                col.bgYellow(
+                    "Registering Loan Protocol Settings Module to LoanTokenLogicBeaconWrbtc..."
+                )
+            );
+
+            data = loanTokenLogicBeaconLMInterface.encodeFunctionData("registerLoanTokenModule", [
+                loanTokenSettingsLowerAdminDeployment.address,
+            ]);
+
+            log(
+                "Generating multisig transaction to replace LoanTokenSettingsLowerAdmin in beacon WRBTC..."
+            );
+            await sendWithMultisig(
+                multisigDeployment.address,
+                loanTokenLogicBeaconWRBTCDeployment.address,
+                data,
+                deployer
+            );
+            log(
+                col.bgBlue(
+                    `>>> DONE. Requires Multisig (${multisigDeployment.address}) signatures to execute tx <<<`
+                )
+            );
+        } else {
+            log(
+                col.bgYellow(
+                    "Skipping LoanTokenSettingsLowerAdmin registration in LoanTokenLogicBeaconWrbtc..."
+                )
+            );
+        }
+
+        /** 5. Replace loan token logic beacon in proxy */
+        loanTokenWithProxyABI = Contract.from_abi(
+            "loanTokenWithProxyABI",
+            (address = loanTokenAddress),
+            (abi = LoanTokenLogicProxy.abi),
+            (owner = conf.acct)
+        );
+        let data = loanTokenLogicBeaconLMInterface.encodeFunctionData("registerLoanTokenModule", [
+            loanTokenLogicLMDeployment.address,
+        ]);
+
+        /** iBPRO */
+        data = loanTokenLogicProxyInterface.encodeFunctionData("setBeaconAddress", [
```

Comment:
pls wrap it into an array/map to make the code consize - same for similar cases in the script

Reply:
updated